src/core/Akka.Remote/EndpointManager.cs

```diff
@@ -340,44 +373,62 @@ protected override SupervisorStrategy SupervisorStrategy()
                 var directive = Directive.Stop;
 
                 ex.Match()
-                    .With<InvalidAddressAssociationException>(ia =>
```

Comment:
Nah, I don't think this can create any issues - the exceptions are never sent over the network themselves, but they're thrown in response to messages (or lack thereof) from the network. Only type of breaking change this could cause would be if someone wrote a RegEx or parser looking for the old exception name specifically, which does show up in error logs.